backend/app/views/spree/admin/stores/_form.html.erb

```diff
@@ -152,6 +152,7 @@
     <%= f.field_container :default_locale, class: ['form-group'] do %>
       <%= f.label :default_locale, Spree.t('i18n.language') %>
       <%= f.select :default_locale, options_from_collection_for_select(all_locales_options, :last, :first, @store.default_locale), {}, { class: 'select2' } %>
+      <%= f.select :default_locale, options_from_collection_for_select(all_locales_options, :last, :first, @store.default_locale || I18n.locale), {}, { class: 'select2' } %>
```

Comment:
@damianlegawiec should we add twice `default_locale` select ?

Reply:
good catch - sth went wrong here ;)